src/pages/Graph/SummaryPanelGraph.tsx

```diff
@@ -95,31 +105,101 @@ export default class SummaryPanelGraph extends React.Component<SummaryPanelPropT
           {this.props.namespaces.map(namespace => namespace.name).join(', ')}
           {this.renderTopologySummary(numSvc, numWorkloads, numApps, numEdges)}
         </div>
-        <div className="panel-body">
-          <div>
-            {trafficRateGrpc.rate > 0 && (
-              <RateTableGrpc
-                title="GRPC Traffic (requests per second):"
-                rate={trafficRateGrpc.rate}
-                rateErr={trafficRateGrpc.rateErr}
-              />
-            )}
-            {trafficRateHttp.rate > 0 && (
-              <RateTableHttp
-                title="HTTP Traffic (requests per second):"
-                rate={trafficRateHttp.rate}
-                rate3xx={trafficRateHttp.rate3xx}
-                rate4xx={trafficRateHttp.rate4xx}
-                rate5xx={trafficRateHttp.rate5xx}
-              />
-            )}
-            {this.shouldShowRPSChart() && (
-              <div>
-                <hr />
-                {this.renderRpsChart()}
-              </div>
-            )}
-          </div>
+        <div
+          className="panel-body"
+          style={{ padding: '0px', paddingLeft: '15px', paddingRight: '15px', paddingBottom: '15px' }}
```

Comment:
nitpick, use `padding: 0 15px 15px;` (top, left-right, bottom) or change `padding: '0px'` to `padding-top: '0px'` to avoid confusions.

Reply:
Also, please move it to typestyle

Reply:
@josejulio This code is unchanged by the PR other than it's placement in the file.  But I'll do it as a tangential refactor.

Reply:
fixed.